research-hub-web/src/app/components/home/home.component.scss

```diff
@@ -45,37 +34,55 @@
   z-index: 1;
 }
 
-/**
- * Contact Section
- */
-.contact-container {
-  z-index: 2;
-  .contact {
-    margin: 0 auto;
+div.feedback-container {
+  text-align: center;
+  @include size(large) {
+    text-align: left;
+  }
+  p {
+    font-size: 16px;
     text-align: center;
     @include size(large) {
       text-align: left;
     }
-    .text {
-      width: 100%;
-      @extend .text;
-      @include size(large) {
-        text-align: left;
-      }
-    }
-    .standard-button {
-      color: #eeeeee !important;
-      padding-bottom: 2.4px !important;
-      border: 3px solid $dark-grey !important;
-      border-radius: 50px !important;
-      background-color: $dark-grey !important;
-      transition: background-color 0.1s ease-in-out, color 0.1s ease-in-out;
-    }
-    .standard-button:hover {
-      color: $dark-grey !important;
-      border-radius: 50px !important;
-      background-color: #eeeeee !important;
-      transition: background-color 0.1s ease-in-out, color 0.1s ease-in-out;
-    }
   }
 }
+
+/**
```

Comment:
commented out stuff - remove?

Reply:
Yes, I tried to find all instances where I left comments, but it seems like I have missed a few :)